client/app/dbaas/logs/logs.html

```diff
@@ -0,0 +1,4 @@
+<div class="cui-page logs">
+    <header ui-view="logsHeader"></header>
```

Comment:
hardcoded values. Will be translated later I guess?

Reply:
This change got into the PR because of merge conflicts. It is not a part of the PR now :)